### DISCUSSION TITLE
element-wise np.maximum/minimum

### DISCUSSION THREAD

**Discussion by nschloe** [data-apis/consortium-feedback#10] [Ideas] [answered]
I sometimes need to compute the maximum or minimum of two arrays. [`np.maximum`](https://numpy.org/doc/stable/reference/generated/numpy.maximum.html) and [`np.minimum`](https://numpy.org/doc/stable/reference/generated/numpy.minimum.html) do that, but I haven't found an equivalent in the API.

**Comment by honno** [comment, marked as answer]
I think as this can be achieved with a mix of `xp.where()` and less/greater ops, it's really a question of would these methods be used enough to justify the maintenance/"API bloat"—I have no idea myself :sweat_smile:

**Reply by kgryte** [reply]
Element-wise `minimum` and `maximum` are decent candidates for standardization, given their broad [support](https://github.com/data-apis/array-api-comparison/blob/a2fdc6b7a7ac4728519785550c6cf25a7ca69973/data/common_apis.csv#L129) across array libraries.
